contracts/RootChain/RootChain.sol

```diff
@@ -64,8 +64,8 @@ contract RootChain {
         public
     {
         authority = msg.sender;
-        currentChildBlock = 1;
-        validatorBlocks = 1;
+        currentChildBlock = 0;
```

Comment:
I'm in favor of keeping blocks 1-indexed. Since position2 in a transaction is zeroed out if we are spending from only 1 UTXO, it makes it easy to check if this is the case by looking at BlkNum2. If we 0-index blocks then the UTXO position [0, 0, 0] is valid, which complicates things.

Reply:
ah, you are right. Committing the change